docs/topics/mta-rn-upgrade-cli.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * docs/web-console-guide/master.adoc
+
+:_content-type: PROCEDURE
+[id="mta-rn-upgrade-cli_{context}"]
+= Setting the parameter in the CLI
+
+You can set the value of `rwx_supported` when you install the {ProductShortName} CLI.
+
+.Procedure
+
+. Follow the installation instructions in the link:{ProductDocUserGuideURL}/index#installing-web-console-or-cli-tool_cli-guide[CLI guide].
+
+. Set `rwx_supported` to `false` by entering the following command:
```

Comment:
I guess applying this is optional

Reply:
@ibragins The procedure is optional, but if the user wants to set this parameter, they must complete this step. Would it help if I changed the sections names' to "Setting the value of rwx_supported in the CLI" and "Setting the value of rwx_supported in the user interface"  ?

Reply:
It cannot be optional here. This section is all about "Support for non-administrators to run MTA" when `rwx_supported: false`